test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -171,6 +171,7 @@ TEST_F(ClusterManagerImplTest, MultipleProtocolCluster) {
       http_protocol_options: {}
       protocol_selection: USE_DOWNSTREAM_PROTOCOL
   )EOF";
+  EXPECT_CALL(factory_.local_info_, node());
```

Comment:
In general, expectations like these don't add much test value since we aren't really testing any actual behavior. Can you instead convert to NiceMock<> so they don't error and then remove these expectations? (Same comment applies everywhere).